presto-mysql/src/test/java/io/prestosql/plugin/mysql/TestMySqlTypeMapping.java

```diff
@@ -219,6 +239,101 @@ public void testDecimalExceedingPrecisionMax()
         testUnsupportedDataType("decimal(50,0)");
     }
 
+    @DataProvider
+    public Object[][] testDecimalExceedingPrecisionMaxProvider()
+    {
+        return new Object[][] {
+                {60, 20},
+                {65, 25},
+        };
+    }
+
+    @Test(dataProvider = "testDecimalExceedingPrecisionMaxProvider")
+    public void testDecimalExceedingPrecisionMaxWithDecimalMapping(Integer typePrecision, Integer typeScale)
+    {
+        JdbcSqlExecutor jdbcSqlExecutor = new JdbcSqlExecutor(mysqlServer.getJdbcUrl());
+
+        List<String> testData = asList(
+                "'exact', 1.12345678",
+                "'exact',-1.12345678",
+                "'round', 12345678901234567890.12345678901234567890",
+                "'round',-12345678901234567890.12345678901234567890",
+                "'unsupported', 1234567890123456789012345678901234567890.1234567890",
+                "'unsupported',-1234567890123456789012345678901234567890.1234567890");
+
+        try (TestTable testTable = new TestTable(
+                jdbcSqlExecutor,
+                "tpch.test_exceeding_max_decimal",
+                format("(label varchar(12), d_col decimal(%d,%d))", typePrecision, typeScale),
+                testData)) {
+            String selectSchema = format("SELECT column_name, data_type FROM information_schema.columns WHERE table_schema = 'tpch' AND table_name = '%s'", testTable.getName().split("[.]")[1]);
```

Comment:
you select columns here not schema. Did you mean column schema aka column data type?